### PR TITLE
Feat/pci deals

### PR DESCRIPTION
**Pull request by marie-j**
ref: MANAGER-4355

Related : 
- [x] 2API
- [x] https://github.com/ovh-ux/ovh-api-services/pull/282

### CODE REVIEW

File: packages/manager/modules/pci/src/components/project/flavor-billing/flavor-billing.html [hunk 1]
```diff
@@ -27,12 +27,28 @@
         data-on-change="$ctrl.selectBilling(modelValue)"
         data-disabled="$ctrl.disabled && $ctrl.monthly"
     >
+        <oui-select-picker-picture>
+            <pci-project-offer-badge
+                data-ng-if="$ctrl.offer"
+                data-title="{{ $ctrl.offer.name.content }}"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/components/project/flavor-billing/flavor-billing.html]: Can it really change at runtime ? (no `::` ?)

File: packages/manager/modules/pci/src/components/project/offer/notice/notice.template.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<small>
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/components/project/offer/notice/notice.template.html]: I'm septical about this, I think the content of the legal text should be in a `small`, I don't think a link to the lega content should, but I'm not sure at all 😄

Comment (FredericEspiau) [reply packages/manager/modules/pci/src/components/project/offer/notice/notice.template.html]: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-small-element

File: packages/manager/modules/pci/src/components/project/offer/notice/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "pci_project_offer_notice": "Mentions légales",
+  "pci_project_offer_notice_new_window": "(S'ouvre dans un nouvel onglet)"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/components/project/offer/notice/translations/Messages_fr_FR.json]: Will the screen reader also read the parenthesis you think ? 😄

File: packages/manager/modules/pci/src/components/project/offer/offer.template.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<div class="d-flex">
+    <pci-project-offer-badge
+        data-title="{{ $ctrl.offer.name.content }}"
+        data-color="{{ $ctrl.offer.name.color }}"
+        data-background="{{ $ctrl.offer.name.background }}"
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/components/project/offer/offer.template.html]: `::`?

File: packages/manager/modules/pci/src/components/project/offer/notice/notice.template.html (hunk 2 above)
Comment (marie-j) [reply packages/manager/modules/pci/src/components/project/offer/notice/notice.template.html]: I'm not sure about it either 😕

File: packages/manager/modules/pci/src/components/project/offer/badge/badge.scss [hunk 5]
```diff
@@ -0,0 +1,6 @@
+pci-project-offer-badge {
```
Comment (radireddy) [inline packages/manager/modules/pci/src/components/project/offer/badge/badge.scss]: you mean .pci-project-offer-badge?

Comment (marie-j) [reply packages/manager/modules/pci/src/components/project/offer/badge/badge.scss]: No `pci-project-offer-badge` is the name of the component so this is meant to scope any style added to this component

File: packages/manager/modules/pci/src/projects/projects.routing.js [hunk 6]
```diff
@@ -22,8 +23,12 @@ export default /* @ngInject */ ($stateProvider) => {
       breadcrumb: /* @ngInject */ () => null,
       confirmDeletion: /* @ngInject */ ($state) => (project) =>
         $state.go('pci.projects.remove', { projectId: project.project_id }),
-      /* @ngInject */
-      defaultProject: (PciProjectsService) =>
+      deals: /* @ngInject */ ($q, OvhApiCloud) =>
+        OvhApiCloud.Aapi()
+          .getDeals()
+          .$promise.then((deals) => new Offer(deals))
+          .catch(() => $q.when({ active: false })),
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/projects.routing.js]:
```suggestion
          .catch(() => ({ active: false })),
```
is enough

**Comment by JeremyDec** [comment]
@marie-j targeting master ?

**Pull request merged**